internal/computeagent/doc.go

```diff
@@ -1 +1,11 @@
+// This package contains the proto and compiled go files for the compute
```

Comment:
Package comments start with the name of the package. "Package computeagent does x..."